.github/workflows/PostCheckList.yaml

```diff
@@ -0,0 +1,36 @@
+name: Post check list
+on:
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened]
+
+jobs:
+  post_check_list:
+    name: Post check list
+    timeout-minutes: 10
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: write
+      checks: write
+      contents: write
+      deployments: write
+      issues: write
+      packages: write
+      pull-requests: write
+      repository-projects: write
+      security-events: write
+      statuses: write
```

Comment:
Is this required?
There seems to be no difference with the default permissions.
ref: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token